compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2922,10 +2922,15 @@ setLoweringConfigForComputeOps(mlir::FunctionOpInterface entryPointFn,
   // loads and stores will have a performance impact.
   auto resultTypes = rootOperation->getResultTypes();
   if (commonVecTileSizes.size() != 0 && !resultTypes.empty()) {
-    auto elementTypeSize =
-        cast<ShapedType>(rootOperation->getResultTypes().front())
-            .getElementType()
-            .getIntOrFloatBitWidth();
+    Type elementType = cast<ShapedType>(rootOperation->getResultTypes().front())
+                           .getElementType();
```

Comment:
This can be `resultTypes.front()`( or `resultTypes[0]` I guess).

```suggestion
    Type elementType = cast<ShapedType>(resultTypes.front())
                           .getElementType();
```